tests/console/consoletest_setup.pm

```diff
@@ -68,7 +68,7 @@ sub run() {
     }
 
     # init
-    select_console 'root-console';
+    check_console_font;
```

Comment:
consoletest_setup is very big and takes long. I get your motivation why you want to check this already at the start but I guess it makes more sense to split the modules then, e.g. mini-setup first, check_console_font second, then next?